### ISSUE TITLE
Peer Review - Unclear Requirements for Accessibility Compliance

### ISSUE THREAD

**Issue by WajdanF** [PKALXI/RapidCare#79]
**Artifact Under Review**
Unclear Requirements for Accessibility Compliance (3.2 and 3.3)

**Team Number for Team Doing the Review**
Team 5

**Description of Issue**
Lacks specific requirements related to compliance with standards such as WCAG (Web Content Accessibility Guidelines) or Section 508. Defining accessibility requirements would ensure the system is usable by healthcare professionals with disabilities, adhering to legal and ethical standards.

**Possible Fix**
Perhaps look into specific types of accessibility guidelines you want to follow. I personally recommend WCAG 2.2 AA (AAA might be a little too time consuming to implement in our given time).
